src/modules/bot/commands/questions/questions-interaction.collector.ts

```diff
@@ -72,19 +79,26 @@ export class QuestionsInteractionCollector {
 
     this.logger.log(`${interaction.user.username} selected a question: "${selectedQuestion.title}"`)
 
-    const watchButton = new ButtonBuilder()
+    this.watchButton = new ButtonBuilder()
       .setCustomId(WATCH_BUTTON)
       .setLabel('İzle')
       .setStyle(ButtonStyle.Primary)
 
-    const listenButton = new ButtonBuilder()
+    this.listenButton = new ButtonBuilder()
       .setCustomId(LISTEN_BUTTON)
       .setLabel('Dinle')
       .setStyle(ButtonStyle.Success)
 
+    this.stopButton = new ButtonBuilder()
+      .setCustomId(STOP_BUTTON)
+      .setLabel('Durdur')
+      .setStyle(ButtonStyle.Danger)
+      .setDisabled(true)
+
     const buttonRow = new ActionRowBuilder<MessageActionRowComponentBuilder>().addComponents(
-      watchButton,
-      listenButton
+      this.watchButton,
+      this.listenButton,
+      this.stopButton
```

Comment:
We should not send the stop button here, as the user has not yet selected any action (watch, listen). I think it would be better to send stop button after user click listen button.